js/client.js

```diff
@@ -2409,7 +2409,7 @@ function toId() {
 	});
 
 	Config.groups = Config.groups || {
-		'~': {
+	'~': {
```

Comment:
```suggestion
		'~': {
```

Blah, that was the wrong number of tabs.